src/NuGet.Core/NuGet.ProjectManagement/Projects/FolderNuGetProject.cs

```diff
@@ -109,12 +103,12 @@ public override Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(Ca
                     await PackageExtractor.ExtractPackageAsync(
                         downloadResourceResult.PackageStream,
                         PackagePathResolver,
-                        nuGetProjectContext.PackageExtractionContext,
-                        PackageSaveMode,
+                        nuGetProjectContext.PackageExtractionContext ?? new PackageExtractionContext(),
```

Comment:
:shipit:

Sent from my Windows Phone
________________________________
From: Deepak Aravindakshan<mailto:notifications@github.com>
Sent: ‎1/‎7/‎2016 1:00 PM
To: NuGet/NuGet.Client<mailto:NuGet.Client@noreply.github.com>
Cc: Yishai Galatzer<mailto:yigalatz@microsoft.com>
Subject: Re: [NuGet.Client] Adds support for PackageSaveMode (#179)


In src/NuGet.Core/NuGet.ProjectManagement/Projects/FolderNuGetProject.cs<https://github.com/NuGet/NuGet.Client/pull/179#discussion_r49125116>:

> @@ -109,12 +103,12 @@ public override Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(Ca
>                      await PackageExtractor.ExtractPackageAsync(
>                          downloadResourceResult.PackageStream,
>                          PackagePathResolver,
> -                        nuGetProjectContext.PackageExtractionContext,
> -                        PackageSaveMode,
> +                        nuGetProjectContext.PackageExtractionContext ?? new PackageExtractionContext(),


@yishaigalatzer<https://github.com/yishaigalatzer>,

I remember why I did this. INuGetProjectContext is an interface. And, setting the default value for PackageExtractionContext needs to happen for every implementation of INuGetProjectContext. Note that there multiple properties of INuGetProjectContext which are null by default.

—
Reply to this email directly or view it on GitHub<https://github.com/NuGet/NuGet.Client/pull/179/files#r49125116>.